src/libs/actions/Timing.js

```diff
@@ -41,8 +41,8 @@ function end(eventName, secondaryNameParam = 'env') {
         }
 
         const grafanaEventName = secondaryName
-            ? `expensify.cash.${eventName}.${secondaryName}`
-            : `expensify.cash.${eventName}`;
+            ? `expensify.${envName}.${eventName}.${secondaryName}`
```

Comment:
@roryabraham - Curious for your thoughts here, we will break old data doing this, but what about:

`${envName}.new.expensify.${eventName}.${secondaryName}` ?

Reply:
yeah we will break the old format 😅 I don't know what format of the event name you guys prefer, we can definitely roll with the one you suggested 👍